.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
are those `.idea` config files really necessary?

Reply:
No :)

Reply:
ok cool, wondered if they contain some nice config you did for IDEA users and then they might have been nice :)